### PR TITLE
(feat) O3-1681: Tiles should have a max width

### PR DESCRIPTION
**Pull request by jnsereko**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary

Tiles had a full width which violates the design https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d59c8c2ad2f809f62880ec 
cc @gracepotma @hadijahkyampeire @denniskigen 

## Screenshots

https://user-images.githubusercontent.com/58003327/216000428-e747b0ee-545d-4c39-9c8b-4c2eb631b21c.mov

## Related Issue

https://issues.openmrs.org/browse/O3-1681

### CODE REVIEW

**Review by hadijahkyampeire — APPROVED** [review]
LGTM, thanks @jnsereko

File: packages/esm-patient-chart-app/src/patient-chart/patient-chart.scss [hunk 1]
```diff
@@ -18,6 +18,8 @@ $actionPanelExpandedOffset: $actionNavOffset+$actionPanelOffset;
   grid-column: 1;
   align-self: start;
   height: 90%;
+  width: calc(100% - 15.625rem);
+  margin: 0 auto;
```
[the remark this answers is not in the document]
Comment (jnsereko) [reply packages/esm-patient-chart-app/src/patient-chart/patient-chart.scss:22]: so i think what @grace meant with the requirement is to neither use `width: max-content` nor 100% but to make the patient chart occupy only its allocated space by design.  

> Tiles need a max width. They should not expand to the full-width of the screen. Please review the [design docs for tiles in OpenMRS here](https://zeroheight.com/23a080e38/p/41d6df-data-tile)

It was hard for me to understand in the first place.  

Actually, assigning the patient-chat a full width messes up the IU, where the visit-actions get hidden introducing a horizontal scroller to cover the patient-chart's whole width

[the remark this answers is not in the document]
Comment (vasharma05) [reply packages/esm-patient-chart-app/src/patient-chart/patient-chart.scss:22]: Yes, but this is a partial understanding of what the ticket is for.

If you check the designs for the patient chart (link in the PR description), you see that the tables expands to the whole screen for tablets, have margins on left and right for small desktop and for large desktops the margins increases, with the screen size, but the width of the patient chart's tables remains the same.

You, on the other hand, you gave the `chart-review` a width of `100% - 15.625rem`, which will break for the tablets.

[the remark this answers is not in the document]
Comment (vasharma05) [reply packages/esm-patient-chart-app/src/patient-chart/patient-chart.scss:22]: For the large desktop, see the widgets are expanding more than they are expected.
image

[the remark this answers is not in the document]
Comment (vasharma05) [reply packages/esm-patient-chart-app/src/patient-chart/patient-chart.scss:22]: The designs show the max-width of 688px which is 43rem.

The widths for different will be:

Tablets: 100%
Small and large desktops: 43rem with equal margins and center aligned always

[the remark this answers is not in the document]
Comment (vasharma05) [reply packages/esm-patient-chart-app/src/patient-chart/patient-chart.scss:22]:
```
width: 100%
max-width: 43rem
```

[the remark this answers is not in the document]
Comment (jnsereko) [reply packages/esm-patient-chart-app/src/patient-chart/patient-chart.scss:22]: hey @vasharma05, kindly f/u at https://github.com/openmrs/openmrs-esm-patient-chart/pull/982

> the large-desktop (2 columns) design is a little confusing.
two buttons control column view in the large desktop with 2 Columns but are absent in large desktop with 1 column.

**Pull request merged**